plugin/webdevicons.vim

```diff
@@ -90,6 +90,31 @@ call s:set('g:DevIconsDefaultFolderOpenSymbol', '')
 " local functions {{{2
 "========================================================================
 
+" scope: local
+function s:getDistro()
+  if exists('s:distro')
+    return s:distro
+  endif
+
+  let s:lsb = system('lsb_release -i')
```

Comment:
Should add executable('lsb_release') before this for Windows.

Reply:
@mattn
Thanks for code review!
I create Pull Request to resolve this problem.